lib/controllers/ActuatorController.h

```diff
@@ -25,12 +25,13 @@ class ActuatorController {
 		float minVelocity_DegreesPerSec = -10.0, maxVelocity_DegreesPerSec = +10.0;
 		float minAngle_Degrees = -90, maxAngle_Degrees = +90;
 
+		float max_force_fsr_newtons = 40; //TODO: Determine this value
```

Comment:
There shouldn't be changes to the ActuatorController class in this PR. ActuatorController is meant to be a generic class for actuators that can be extended, FSR is a specific thing for a single mechanism, the claw